packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -119,8 +119,6 @@ export function SelectBase<T>({
   maxVisibleValues,
   menuPlacement = 'auto',
   menuPosition,
-  // TODO change this to default to true for Grafana 9
-  menuShouldPortal = false,
```

Comment:
We're just not allowing opt-out any more at all?

Reply:
guess we can discuss here 😄  i'd love to remove it completely tbh, but it might be wise to have a safety hatch just in case. anyone else have thoughts?